visualization/app/codeCharta/ui/screenshotButton/screenshotButton.component.spec.ts

```diff
@@ -45,20 +49,37 @@ describe("resetSettingsButtonController", () => {
 			threeSceneService,
 			threeCameraService,
 			threeRendererService,
-			storeService
+			storeService,
+			$rootScope
 		)
 	}
 
 	describe("makeScreenshot", () => {
```

Comment:
Do we have a test case for `makeScreenshotToClipboard()`?

Reply:
not yet

Reply:
i wrote one now